src/schema/assets/image.rs

```diff
@@ -42,7 +42,7 @@ where
 }
 
 // Function signature must match Serde's Serialize trait, so need to suppress Clippy.
-#[allow(clippy::trivially_copy_pass_by_ref)]
+#[expect(clippy::trivially_copy_pass_by_ref, reason = "Deferred")]
```

Comment:
The reason is present in the comment above.

```suggestion
#[expect(clippy::trivially_copy_pass_by_ref, reason = "Function signature must match Serde's Serialize trait")]
```